md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/CodeUploaderAPI.java

```diff
@@ -158,6 +158,17 @@ RestWrapper zipUpload(   //This is lib, hql ,zip etc according to file to be upl
                 stream.write(bytes);
                 stream.close();
                 //Populating Uploaded file bean to return in RestWrapper
+                LOGGER.info("name of the file is + "+name);
+                if (CodeUploaderAPI.isNumeric(subDir))
+                {
+                    File oldName = new File(uploadLocation+"/"+name);
+                    File newName = new File(uploadLocation+"/"+"analytic.png");
+                    if(oldName.renameTo(newName)) {
+                        System.out.println("renamed");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS106)